bundles/binding/org.openhab.binding.mqtt/README.md

```diff
@@ -21,18 +25,19 @@ This binding can be configured in the file `services/mqtt.cfg`.
 
 In order to consume or publish messages to an MQTT broker, you need to define all the brokers which you want to connect to, in your `services/mqtt.cfg` file.
 
-| Property | Default    | Description |
-|----------|------------|-------------|
-| `<broker>`.url |      | Required. URL to the MQTT broker, e.g. tcp://localhost:1883 or ssl://localhost:8883 |
-| `<broker>`.clientId | | Client id (max 23 chars) to use when connecting to the broker. If not provided a default one is generated. |
-| `<broker>`.user |     | User ID to authenticate with the broker. |
-| `<broker>`.pwd |      | Password to authenticate with the broker. |
-| `<broker>`.qos | 0    | Set the quality of service level for sending messages to this broker. Possible values are 0 (Deliver at most once),1 (Deliver at least once) or 2 (Deliver exactly once). |
-| `<broker>`.retain | false | true or false. Defines if the broker should retain the messages sent to it. |
-| `<broker>`.async | true | true or false. Defines if messages are published asynchronously or synchronously. |
-| `<broker>`.keepAlive | 60 | Integer. Defines the keep alive interval in seconds. |
+| Property             | Default | Required | Description                                                 |
+|----------------------|---------|:--------:|-------------------------------------------------------------|
+| `<broker>`.url       |         | Yes      | URL to the MQTT broker, e.g. tcp://localhost:1883 or ssl://localhost:8883 |
+| `<broker>`.clientId  |         | Yes/No   | Client ID to use when connecting to the broker. If not provided a random default is generated. Usually restricted to 23 characters in length. (see the `allowLongerClientIds` setting) |

```

Comment:
I think Yes* is a better description, since it's required but has a random default if not specified by the user (see latest push). WDYT?